src/components/NavTop.js

```diff
@@ -29,30 +13,38 @@ const StyledH2 = styled.h2`
   font-style: none;
 `;
 
-const LeaveSettings = styled(Cross)`
-  height: 35px;
-  width: 35px;
+const SettingsInput = styled.button`
```

Comment:
The name `SettingsInput` isn't that clear to me. For me it sounds like, that i can type something into an input field on the settings-page

Reply:
I´ll definitely have a look at this :).